docs/lib/predictions/fragments/android/escapehatch.md

```diff
@@ -0,0 +1,47 @@
+If you need functionality in the AWS services used by the Amplify Predictions category that isn't available, we provide an escape hatch so you can get a reference to that service. For example, to get a reference to `AmazonRekognitionClient`:
+
+```java
+// Obtain reference to the plugin
+final AWSPredictionsPlugin predictionsPlugin;
```

Comment:
Hmm, instead of setting the `AWSPredictionsPlugin` externally, why not declare the `AWSPredictionsEscapeHatch` and set it in the try/catch?

Reply:
`try-catch` should not even be necessary, but I only had it to match the error message on iOS side. This will only break if the predictions plugin was not configured yet.

Reply:
Ah! I think that would be self-explanatory. I'd remove this for clarity.